app/templates/partials/section.html

```diff
@@ -1,6 +1,6 @@
 <div class="section" id="{{section.id}}">
   {% if section.schema_item.title %}
-    <h1 class="section__title saturn">{{section.schema_item.title}}</h1>
+    <h1 class="section__title saturn"><span>{{section.schema_item.number}}. </span>{{section.schema_item.title}}</h1>
```

Comment:
Section number is optional, so we shouldn't output the span if it's not in the schema.